apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsImageControl.tsx

```diff
@@ -71,6 +78,11 @@ export function JsonFormsImageControl({
         <Text textStyle="body-2" textColor="base.content.medium" pt="0.5rem">
           {`Maximum file size: ${MAX_IMG_FILE_SIZE_BYTES / 1000000} MB`}
         </Text>
+        {!!errors && (
+          <FormErrorMessage>
+            {label} {errors}
```

Comment:
assuming errors is a string here

Reply:
yeap, that's the type given by json forms